plugin/plaintextmodel.cpp

```diff
@@ -116,10 +122,16 @@ void PlainTextModel::setSource(const QUrl &source)
                 qmlInfo(this) << "Can't open " << m_source << ": " << m_file.errorString();
                 m_status = Error;
             } else {
+                uchar * res = m_file.map(0, m_file.size());
```

Comment:
We should limit the size here I think, something like `min(m_file.size(), 5*1024*1024)` for instance.